AnaProd/anaTupleProducer.py

```diff
@@ -115,12 +119,24 @@ def addAllVariables(dfw, syst_name, isData, trigger_class):
                                     f"tau{leg_idx+1}_recoJetMatchIdx>=0 ? static_cast<float>(Jet_p4.at(tau{leg_idx+1}_recoJetMatchIdx).Phi()) : -1.f;")
         dfw.DefineAndAppend( f"tau{leg_idx+1}_seedingJet_mass",
                                     f"tau{leg_idx+1}_recoJetMatchIdx>=0 ? static_cast<float>(Jet_p4.at(tau{leg_idx+1}_recoJetMatchIdx).M()) : -1.f;")
+        dfw.DefineAndAppend( f"tau{leg_idx+1}_seedingJet_e",
+                                    f"tau{leg_idx+1}_recoJetMatchIdx>=0 ? static_cast<float>(Jet_p4.at(tau{leg_idx+1}_recoJetMatchIdx).E()) : -1.f;")
```

Comment:
```suggestion
```

Reply:
maybe I can remove this, but I prefer for the moment to store also energies.